### ISSUE TITLE
SC2253 error missing from README.md

### ISSUE THREAD

**Issue by roymarantz** [hadolint/hadolint#372]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
I would find documentation for SC2253 like other errors

### Actual behavior
nothing found in README.md

### Steps to reproduce the behavior
search for that string in README.md

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
# installed via brew install hadolint a few minutes ago
$ hadolint --version
Haskell Dockerfile Linter 1.17.2-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**

**Comment by zemanlx** [comment]
I am sorry, but a list of errors is not exhaustive as it is mentioned in README. However, feel free to make a PR with a description of more issues.
